bots/psuteam7bot/castle.js

```diff
@@ -13,30 +13,111 @@ castle.doAction = (self) => {
     {
         return castle.findUnitPlace(self, 'PILGRIM');
     }
-    else
+    else if(self.me.turn<100)
```

Comment:
Probably should not merge this if conflict - don't want to stop building after turn 100